proto/v2/observation.proto

```diff
@@ -56,17 +65,17 @@ message ObservationRequest {
   // - "<DATE>": a speficied valid ISO 8601 date. Observation corresponding to
   //   this date is returned.
   string date = 5;
-  // When specified, only observation with this unit is returned.
-  string unit = 6;
-  // When specified, only observation with this measurement method is returned.
-  string measurement_method = 7;
-  // When specified, only observation with this observation period is returned.
-  string period = 8;
+  // Value of the observation
+  string value = 6;
+  // [Optional] filter for the facet
+  FacetFilter filter = 7;
```

Comment:
If we are OK having nested protos for request, I slightly wonder if we want to do a common message for variable and entity:

message ExpIdArg {
  repeated dcids = 1;
  string expression = 2;
}
ExpIdArg entity = 1;
ExpIdArg variable = 2;

Then we can use singular and make it consistent with filter (to Wei's point)

Reply:
Both suggestions are very good. Updated